cscs-checks/mch/src/openacc_cuda_mpi_cppstd.F90

```diff
@@ -74,9 +74,9 @@ program openacc_cuda_mpi_cppstd
     if (sum(f1) /= EXPECTED_CUDA_SUM) then
       write (*,*) "Result : FAIL"
       write (*,*) "Expected value sum(f1): ", EXPECTED_CUDA_SUM, "actual value:", sum(f1)
-    else if (sum(f3) /= EXPECTED_CPP_STD_SUM) then
+    else if (sum(f2) /= EXPECTED_CPP_STD_SUM) then
```

Comment:
no it should be f3, but the error is on the right side

Reply:
like this:

else if (sum(f3) /= EXPECTED_CUDA_SUM) then

?

Reply:
Yes, basicall revertting the change from PR #888 only the file openacc_cuda_mpi_cppstd.F90  . I did that with Carlos, because the code was failling. But probably because we did too quickly and did not compile with the correct OpenACC flag. Note that you also need to adapt the 
```
 write (*,*) "Expected value sum(f3): ", EXPECTED_CUDA_SUM, "actual value:", sum(f3)
```
below

Reply:
sorry for messing this up